lib/offline.js

```diff
@@ -58,14 +61,48 @@ module.exports = promisify(function(config, callback) {
       return '';
     }
   }).then(function(cacheId) {
-    swPrecache.write(path.join(rootDir, 'offline-worker.js'), {
-      staticFileGlobs: fileGlobs.map(function(v) { return path.join(rootDir, v) }),
-      stripPrefix: rootDir,
-      verbose: true,
-      logger: gutil.log,
-      importScripts: config.importScripts || [],
-      cacheId: cacheId,
-      ignoreUrlParametersMatching: config.ignoreUrlParametersMatching || [/./],
-    }, callback);
+    var absoluteGlobs =
+      fileGlobs.map(function (v) { return path.join(rootDir, v); });
```

Comment:
Note that there's no line width restriction for this codebase. Statements like this are welcome to live on one line and exceed 80 columns:

```js
    var absoluteGlobs = fileGlobs.map(function (v) { return path.join(rootDir, v); });
```

(That said, it's also ok to wrap lines; do whatever feels most readable!)